pom.xml

```diff
@@ -1714,6 +1714,11 @@
     </dependencies>
   </dependencyManagement>
   <repositories>
+    <repository>
+      <id>nexus-aliyun</id>
+      <name>Nexus aliyun</name>
```

Comment:
do we need this?

Reply:
Sorry, it was too late at the time and I didn't pay much attention.